app/not-found.tsx

```diff
@@ -1,13 +1,24 @@
-import Link from 'next/link';
+'use client';
+
+import NotFoundAnimation from '../public/lottiefiles/404.json';
+import { Player } from '@lottiefiles/react-lottie-player';
 
 export default function NotFound() {
+  
   return (
-    <div className="flex flex-col m-auto">
-      <h2 className="font-bold flex flex-col m-auto">Error 404</h2>
-      <p>Oops! This page was not found</p>
-      <button className="border-2">
-        <Link href="/">Return Home</Link>
-      </button>
+    <div>
+      <h2 className="mt-12 text-3xl font-semibold">Oops! Page Not Found</h2>
+      <p className="mt-4">We couldn&apos;t find the page you&apos;re looking for</p>
+      <div aria-label='Page not found' aria-live='assertive'>
+        <Player
+        autoplay={true}
+        loop={false}
+        src={NotFoundAnimation}
+        className="w-[350px] h-[350px]"
+        >
+        </Player>
+      </div>
```

Comment:
Maybe move `click` outside of the hyperlink text to clarify `here` is the word to interact with for navigation?
```
<p className="mb-64 text-light-text">Please click<a href="/" className="text-light-text underline hover:via-light-hover">here</a> to return home or contact us at <a href="mailto:info@linkta.org">info@linkta.org</a></p>
```